UI/package.json

```diff
@@ -15,6 +15,7 @@
     "@mdi/font": "5.9.55",
     "@tailwindcss/postcss7-compat": "2.1.2",
     "adal-angular": "1.0.18",
+    "arr-flatten": "^1.1.0",
```

Comment:
Where is this library used? We may be able to use lodash which is already included in package.json.